prow/cmd/crier/README.md

```diff
@@ -61,16 +61,16 @@ The **OAuth Access Token** can be obtained as follows:
 1. Click **Create New App**.
 1. Provide an **App Name** (e.g. Prow Slack Reporter) and **Development Slack Workspace** (e.g. Kubernetes).
 1. Click **Permissions**.
-1. Add the `chat:write:bot` scope using the **Scopes** dropdown and **Save Changes**.
+1. Add the `chat:write.public` scope using the **Scopes / Bot Token Scopes** dropdown and **Save Changes**.
```

Comment:
Are you sure this is needed?

Reply:
Without the .public permission, crier got `not_in_channel` as a response. And the Slack dropdown does not contain a `:bot` permission anymore.